test/test_helper.rb

```diff
@@ -181,6 +181,9 @@ def create_webauthn_credential
     fullscreen_headless_chrome_driver
 
     visit sign_in_path
+
+    assert page.has_content?("Sign in")
```

Comment:
My understanding is that we're relying on the implicit `wait` from `page.has_content?`. Is that right? If that's the case, what do you think about adding a code comment explaining why we're putting it there?